src/components/overview/chart/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { colors } from '../../../theme';
+import { data } from './helper';
+import {
+  Area,
+  AreaChart,
+  CartesianGrid,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import { FunctionComponent, useState } from 'react';
+const Chart = () => {
+  const currentDate = new Date();
+  const currentHour = currentDate.getHours();
+  const [isTodayHovered, setIsTodayHovered] = useState(false);
+  const [isYesterdayHovered, setIsYesterdayHovered] = useState(false);
+  interface IProps {
+    cx: number;
+    cy: number;
+    gray?: boolean;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
there is a problem with cx and cy that you don't pass into the `CustomizedDot` below. You need to add those as optional and fix another issue, or pass cx and cy

Reply:
fix: cx & xy made optional and then checked for beiing undefined